internal/orchestrator/updates.go

```diff
@@ -324,6 +325,11 @@ func (o *Orchestrator) eventUpdate(ctx context.Context, evt *v1types.ConditionUp
 
 		// queue any follow-on work as required
 		active, err := o.repository.GetActiveCondition(ctx, evt.ConditionUpdate.ServerID)
+		if err != nil && errors.Is(err, store.ErrConditionNotFound) {
```

Comment:
Its a bit misleading when I read this - should we have two error types here,
`ErrConditionNotFound`, `ErrNoActiveCondition` so they can be explicit

Reply:
While I do value explicitness here, in the end `ErrConditionNotFound` (which arguably should be `ErrNoActiveConditionFound` but... ) wind up being treated exactly the same. It's an error if you were expecting an active condition and not if you weren't. As far as the name goes, I'm ambivalent. We can change it if you think it's useful but I don't see it as adding a bunch of friction to further development or maintenance.

Reply:
fair, it can stay unless it comes up again